ddtrace/contrib/pylons/middleware.py

```diff
@@ -41,7 +41,11 @@ def _start_response(status, *args, **kwargs):
             except Exception as e:
                 # "unexpected errors"
                 # exc_info set by __exit__ on current tracer
-                span.set_tag(http.STATUS_CODE, getattr(e, 'code', 500))
+                code = getattr(e, 'code', 500)
+                if isinstance(code, int) and code >= 100 and code < 600:
```

Comment:
what happen if `e.code = '512'`? because it's not an `int`, it will be wrongly set to `500` right?